File: poo-aulas/03.11/Bhaskara.java
public class Bhaskara {
private double a, b, c;

Bhaskara(double a, double b, double c) {
  this.a = a;
  this.b = b;
  this.c = c;
}

public double calculaX1() {

   double delta = (b*b)-4*(a)*(c);

return (-b + Math.sqrt(delta) ) / (2*a);
}

public double calculaX2() {
   
       double delta = (b*b)-4*(a)*(c);
    
    
return (-b - Math.sqrt(delta) ) / (2*a);
}

public static void main (String args[]) {
    Bhaskara bascara = new Bhaskara(2,-5,-7);
System.out.println(bascara.calculaX1());
System.out.println(bascara.calculaX2());
}
}

File: 03.18/Bhaskara.py
import math

class Bhaskara:
    def __init__(self, a, b, c):
        self.a = a
        self.b = b
        self.c = c

    def calcula_x1(self):
        delta = (self.b ** 2 - 4 * self.a * self.c)
        return (-self.b + math.sqrt(delta)) / (2 * self.a)

    def calcula_x2(self):
        delta = (self.b ** 2 - 4 * self.a * self.c)
        return (-self.b - math.sqrt(delta)) / (2 * self.a)

print("Aluno: Brunno Cunha Bernardes")
print("Matrícula: 202405846")

bascara = Bhaskara(1, -6, 5)

print("X1 =", bascara.calcula_x1())
print("X2 =", bascara.calcula_x2())
